### DISCUSSION TITLE
Memory error when executing occ

### DISCUSSION THREAD

**Discussion by raspberryenvoie** [nextcloud/all-in-one#1917] [Questions] [answered]
### Steps to reproduce
1. Open a console for the container nextcloud-aio-nextcloud
2. Run `sudo -u www-data ./occ`

### Expected behavior 
The command is expected to be successful.

### Actual behavior 
The command outputs:
```Fatal error: Allowed memory size of 2097152 bytes exhausted (tried to allocate 491520 bytes) in /var/www/html/3rdparty/composer/autoload_real.php on line 35```

### Host OS 
Debian GNU/Linux

#### Nextcloud AIO version 
v4.2.0
#### Current channel 
latest

Thanks helping.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#how-to-run-occ-commands
